### DISCUSSION TITLE
Optimize the height of the dashboards list in a directory

### DISCUSSION THREAD

**Discussion by madhead** [grafana/grafana#42103] [General]
**Steps to reproduce**:

Open any directory with a lot of dashboards in Grafana.

**Actual Result**:

The scrollable list of dashboards consumes only 41% of the height (373 out of 901 pixels, to be precise):

![image](https://user-images.githubusercontent.com/577360/142901210-7eb78ec6-1d15-47ad-844c-24d2a277deea.png)

**Expected Result**

The list of dashboards, the most important part of this screen, should definitely have more space.

**Environment**:
- Grafana version:  v8.1.2 (103f8fa094)
- Data source type & version: N/A
- User OS & Browser: Windows 10, Firefox 94.0.1.
- Screen resolution: 1920 × 1080, 125% scale in Windows settings
  
  ![image](https://user-images.githubusercontent.com/577360/142901987-4806ccc7-35a8-4bf1-8040-98fe52ef4c91.png)

**Comment by madhead** [comment]
Probably search, sort, and filter by tags could be combined to take less vertical space. Vertical paddings & margins in general could be decreased, couldn't they?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
